e2e/starter.test.js

```diff
@@ -0,0 +1,19 @@
+describe('Example', () => {
```

Comment:
Why don't you need to import `device` as in the other test?

Reply:
Updated